pkg/omf/cli/omf.install_package.fish

```diff
@@ -1,5 +1,11 @@
-function omf.install_package
+function omf.install_package -d "Return success for at least one successful install."
+  set -l return_code 1
   for search in $argv
     omf.install --pkg $search
+    set -l code $status
+    if test $code -eq 0
```

Comment:
No because as I had in the description I want to return 0 when at least one omf.install succeed.
I'll push another commit to make it clearer.